### ISSUE TITLE
Change how UI shows compound fields

### ISSUE THREAD

**Issue by kamil386** [IQSS/dataverse#6589]
Actually Dataverse compound fields are "flatten" in Dataverse UI:

![image](https://user-images.githubusercontent.com/56053188/73354689-9923a880-4296-11ea-85ae-2f92e2975587.png)

Please add more detailed output of compund fields, at least include title of fields in the UI.

**Comment by scolapasta** [comment]
@kamil386 It is possible to do this via the displayFormat column of the datasetFieldType table (and therefore in the .tsv file that defines the blocks). See for example "Time Period Covered" that adds "Start:" and "End:".

In specific regard to Geographic Coverage, we did also make a recent change that comma separates these values. This is available in 4.19.

**Comment by pdurbin** [comment]
> In specific regard to Geographic Coverage, we did also make a recent change that comma separates these values. This is available in 4.19.

@kamil386 if you're interested in the discussion that took place on this, please see #6345.

**Comment by kamil386** [comment]
@scolapasta It's not enough to format it via displayFormat column in TSV, especially for the big schema like DwC where i.e one parent field has 21 subfields, and another has 44

@pdurbin So I will continue the discussion on #6345

**Comment by kamil386** [comment]
One note, there is no issue while editing big schema like DwC:

![image](https://user-images.githubusercontent.com/56053188/73449739-23860e00-4364-11ea-8c2a-3c17ee603803.png)

But the issue is how it is printed out:

![image](https://user-images.githubusercontent.com/56053188/73449789-487a8100-4364-11ea-8b46-46a175e18460.png)

Maybe is it worth consider to print in different way, including headers and create some table?

**Comment by jggautier** [comment]
I agree, that's hard to read.

Just suggesting another idea using the displayFormat column in the TSV. Would it be possible to add a `<br>` tag at the end of each value so that there's a line break and each subfield would be on its own line?:

1234
Catalog Number: 1234
Record Number: 1234
Recorded by: Kamil
...

**Comment by kamil386** [comment]
Excellent idea, simple and straightforward. I'll check now if it's working as expected. I'll also try to add bold to headers. In the meantime I could suggest adding hints on headers which is now not possible via displayFormat.

**Comment by kamil386** [comment]
Unfortunately, it's not working as expected

![image](https://user-images.githubusercontent.com/56053188/73538944-8e028100-442c-11ea-97ef-8154791471a5.png)

**Comment by wfenrich** [comment]
Also be careful with labels: currently you will probably get "Occurene Occurence Status" or "Occurence Occurence Remarks" in advanced search and facets.

**Comment by jggautier** [comment]
This issue is mainly about how the things people enter in child fields are displayed in the user interface. But @wfenrich's last comment about the display of names of child fields is something that we hope to resolve as part of https://github.com/IQSS/dataverse/issues/8127, which includes writing and using guidelines for naming fields.

**Comment by jggautier** [comment]
This issue came up again when using a metadatablock for a specific collection in the Harvard Dataverse Repository.

The "Study Design" field is made up of four child fields, one of which expects a lot of text:

<kbd><img width="905" alt="Screen Shot 2022-05-16 at 8 42 53 PM" src="https://user-images.githubusercontent.com/18374574/168704858-dafc92be-4eee-4908-ae3e-36c0539f2cad.png"></kbd>

When the metadata is displayed, it's pretty hard to parse, even when :

<kbd><img width="1155" alt="Screen Shot 2022-05-16 at 8 42 32 PM" src="https://user-images.githubusercontent.com/18374574/168704923-d3cf3521-3d3f-44f5-9a23-c3fc01486e98.png"></kbd>

Luckily, as this "Study Design" field doesn't need to be repeatable, the child fields don't need to be kept together as a compound field, so we're planning to separate the child fields into their own fields (and get rid of the "Study Design" "field"). That way what's entered in those four fields are on their own lines when displayed.

But I think this illustrates that compound fields don't even need 44 or 21 subfields to run into this design problem. It's apparent when a compound field has just one field that uses a large text box and multiple lines of text are entered into it.

**Comment by cmbz** [comment]
2024/09/30: Closed as duplicate to https://github.com/IQSS/dataverse/issues/9675
